datahub-web-react/src/app/searchV2/searchBarV2/components/SearchBarInput.tsx

```diff
@@ -110,8 +145,8 @@ const SearchBarInput = forwardRef<InputRef, Props>(
                         <SuffixWrapper>
                             {(showCommandK && !isDropdownOpened && !isFocused && <CommandK />) || null}
                             {viewsEnabled && (
-                                <ViewSelectContainer onClick={(e) => e.stopPropagation()}>
-                                    <ViewSelect />
+                                <ViewSelectContainer onClick={onViewSelectContainerClickHandler}>
+                                    <ViewSelect isOpen={isViewsSelectOpened} onOpenChange={onViewsClickHandler} />
```

Comment:
i'm seeing a few bugs around building views with this new search bar - i'm not sure if they were introduced in this PR or a different one.

regardless, I'm tracking them and will knock them out but want to merge this first so we can unblock the OSS merge